src/test/java/LoginTests.java

```diff
@@ -3,23 +3,22 @@
 import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import pages.BasePage;
+import pages.HomePage;
+import pages.LoginPage;
 
 import java.time.Duration;
 
-public class LoginTests extends BaseTest {
+public class  LoginTests extends BaseTest {
+
     @Test
-    public void LoginEmptyEmailPasswordTest() {
+    public void LoginValidEmailPasswordTest() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public void loginEmptyEmailPasswordTest ()`